### ISSUE TITLE
Invalid File message

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/platform-ui#902]
Currently the error message when a file is named incorrectly is: 

![image](https://user-images.githubusercontent.com/15824252/70647104-3c9cf900-1c16-11ea-897a-5a274f523328.png)

**Change this to:**

Improperly named files cannot be uploaded or validated. All files must start with the corresponding entity name and have the .tsv extension (e.g. donor<_optional_extension>.tsv).

**Comment by wajiha-oicr** [comment]
@kcullion suggestion, would it be more clear if the error message said 

Improperly named files cannot be uploaded or validated. All files must start with the corresponding entity name and have the **correct** .tsv extension (e.g. donor<_optional_extension>.tsv).

Instead of

Improperly named files cannot be uploaded or validated. All files must start with the corresponding entity name and have the .tsv extension (e.g. donor<_optional_extension>.tsv).

?

**Comment by kcullion** [comment]
@wajiha-oicr 

I think saying "have the .tsv extension" means that it can't have a .jpg or .xls extension, for example. If we want to say "correct" then it would probably say something like "have the correct extension" (which would be .tsv). 

Am I understanding correctly? or do you want to say "correct" because of the optional extension they can put on?

**Comment by wajiha-oicr** [comment]
> @wajiha-oicr
> 
> I think saying "have the .tsv extension" means that it can't have a .jpg or .xls extension, for example. If we want to say "correct" then it would probably say something like "have the correct extension" (which would be .tsv).

Ohhhh, I didn't interpret it that way at first but that makes sense!

**Comment by kcullion** [comment]
thanks @wajiha-oicr . yeah maybe it's a bit wordy. thanks for your suggestion though!

**Comment by kcullion** [comment]
@wajiha-oicr maybe we say something like this:

Improperly named tsv files cannot be uploaded or validated. Please retain the template file name and only append characters to the end. (e.g. donor<_optional_extension>.tsv).

**Comment by rosibaj** [comment]
@kcullion do you mind testing this one please!

**Comment by kcullion** [comment]
@rosibaj @wajiha-oicr this looks good!
